specification/index.md

```diff
@@ -1,32 +1,39 @@
-BIDS Stats Models
-=================
+# BIDS Stats Models
 
-*A specification for neuroimaging statistical models.*
+_A specification for neuroimaging statistical models._
 
-*BIDS Stats Models* (BSM) describe how to fit *statistical models for neuroimaging data* using *a machine-readable JSON document*.  Its purpose is prescriptive (e.g. a recipe), rather than descriptive (e.g. summary of already fit models). 
+_BIDS Stats Models_ describe how to fit _statistical models for neuroimaging
+data_ using _a machine-readable JSON document_. Its purpose is prescriptive
+(meaning that it is a recipe), rather than descriptive (as in a summary of
+already fit models).
 
-A *BSM*—in combination with a preprocessed BIDS dataset—is sufficient to execute statistical models, with minimal configuration and intervention required from the user.
+A _BIDS Stats Models_ —in combination with a preprocessed BIDS dataset— is
+sufficient to execute statistical models, with minimal configuration and
+intervention required from the user.
 
-The **mission** of *BSM* is to promote scientific best practice by making neuroimaging models *transparent and reproducible*, as well as encourging the development of *standardized and automated* statistical analysis pipelines.
+The **mission** of _BIDS Stats Models_ is to promote scientific best practice by
+making neuroimaging models _transparent and reproducible_, as well as encourging
+the development of _standardized and automated_ statistical analysis pipelines.
+
+:::{panels} :column: col-lg-6 px-2 py-2
 
-:::{panels}
-:column: col-lg-6 px-2 py-2
 ---
-**📖 Learn more**
-^^^
-**[](motivation.md)**: Why do we need *BSM*?
+
+**📖 Learn more** ^^^ **[](motivation.md)**: Why do we need _BIDS Stats Models_?
```

Comment:
seems the auto lint may have messed up a few things

let me cross check and fix

Reply:
fixed